shared/params/checktags_test.go

```diff
@@ -0,0 +1,11 @@
+// +build !develop
```

Comment:
Can you add a small note in `DEPENDENCIES.md` that you need this tag to run tests in pure go ? Otherwise users building and testing on go might find this an abrupt change.

Reply:
Done!